catalog/datasets/wikidata-entities-all.jsonld

```diff
@@ -15,13 +15,13 @@
   ],
   "distribution": [
     {
-      "@id": "https://www.wikidata.org/sparql",
+      "@id": "https://www.wikidata.org/sparql/entities#all",
```

Comment:
The old URI didn't resolve (I should have taken care of that). Perhaps we can change the new URI to a resolvable one, e.g. `https://query.wikidata.org/sparql#entities-all`?

Reply:
It not really clear to me what is going on here (and in catalog.test.ts) and what the preferred end situation should be like. Should I change the distribution @id in all wikidata dataset definitions or only here? Or maybe have a chat about it?